src/main/kotlin/no/nav/syfo/behandler/BehandlerService.kt

```diff
@@ -178,4 +184,17 @@ class BehandlerService(
             connection.commit()
         }
     }
+
+    private fun updateBehandlerToArbeidstaker(
+        behandlerArbeidstakerRelasjon: BehandlerArbeidstakerRelasjon,
+        behandlerId: Int,
+    ) {
+        database.connection.use { connection ->
+            if (connection.updateBehandlerArbeidstakerRelasjon(behandlerArbeidstakerRelasjon, behandlerId)) {
+                connection.commit()
+            } else {
+                throw RuntimeException("Expected update to single behandlerArbeidstakerRelasjon")
```

Comment:
Noen spesiell grunn til å ha if else og kaste exception herfra i stedet for fra database-laget (som vi gjør ved andre creates/updates) ?

Reply:
Enig, har fikset 👍